src/views/FoldersView.tsx

```diff
@@ -12,27 +12,38 @@ export function FoldersView({ onClick }: { onClick?: (fileEntry: FileEntry) => v
   const [selectedFile, setSelectedFile] = useState<FileEntry>();
 
   useEffect(() => {
-    readAllProjectFiles().then((files) => {
-      setFiles(files);
-      const selected = files.find((f) => f.name?.endsWith('.tiptap'));
-      if (selected) {
-        setSelectedFile(selected); // We need this because we might be selecting a DOT_FILE
-        onClick?.(selected);
-      }
-    });
+    readAllProjectFiles()
+      .then((newFiles) => {
+        setFiles(newFiles);
+        const selected = newFiles.find((f) => f.name?.endsWith('.tiptap'));
+        if (selected) {
+          setSelectedFile(selected); // We need this because we might be selecting a DOT_FILE
+          onClick?.(selected);
+        }
+      })
+      .catch((e) => {
```

Comment:
This is [@typescript-eslint/no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises).